x-pack/docs/en/security/enroll-nodes.asciidoc

```diff
@@ -1,5 +1,27 @@
 [role="exclude"]
 
+When {es} starts for the first time, the security auto-configuration process
+binds the HTTP layer to both `_site_` and `_local_`, but only binds the
+transport layer to `_local_`. This intended behavior ensures that you can start
+a single-node cluster with security enabled by default without any additional
+configuration.
+
+Before enrolling a new node, additional actions such as binding to an address
+other than `localhost` or satisfying bootstrap checks are typically necessary
+in production clusters. During that time, an auto-generated enrollment token
+could expire, which is why enrollment tokens aren't generated automatically.
+
+Additionally, only nodes on the same host can join the cluster without
+additional configuration. If you want nodes from another host to join your
+cluster, you need to set `transport.host` to `_site_`. Refer to 
```

Comment:
```suggestion
cluster, you need to set `transport.host` to something other than `_local_` (i.e. `_site_`, or an IP address bound to an interface that is reachable by other hosts). Refer to 
```

Reply:
Thanks @jkakavas! I like your suggestion. Are you ok with the following change?

>If you want nodes from another host to join your cluster, you need to set `transport.host` to a supported value other than `_local_` (such as `_site_`), or an IP address that's bound to an interface where other hosts can reach it.